.travis.yml

```diff
@@ -37,8 +37,8 @@ install:
   - printf '%s\n\n' '#!/bin/bash' 'echo "Running elm-make with sysconfcpus -n 2"' '$TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 elm-old "$@"' > $(npm config get prefix)/bin/elm
   - chmod +x $(npm config get prefix)/bin/elm
   - npm install
-  - pushd tests && elm make --yes && popd
+  - pushd tests && elm make && popd
```

Comment:
Again, why this revert?

Reply:
Ah, probably because 0.19 `elm make` doesn't have --yes anymore. Fair enough! :+1: